src/_Core.scss

```diff
@@ -229,8 +229,8 @@ body {
   text-align: center;
   justify-content: center;
   text-decoration: none;
-  padding: 0px 25px !important;
-  margin: 0px !important;
```

Comment:
@nickgros , do you know why we need the !important here? I deleted them but if it's going to break something in Portals, please let me know. Thanks.

Reply:
I have no idea why it's used. It looks like the .nav-button class is only used in the navbar, so if your changes look good, then I think you can safely remove them

Reply:
I have no idea why it was originally there. Seems like these styles apply to all of the `nav-button`s just fine without `!important`, so I think it's safe to remove.